docs/ios-installation.md

```diff
@@ -14,9 +14,17 @@ In addition to the standard React Native requirements, you will also need the fo
 
 ## Install react-native-azurenotificationhub
 
-```
-npm install react-native-azurenotificationhub
-```
+Install the library using either **Yarn**:
+
+    yar add react-native-azurenotificationhub
```

Comment:
Do you mean `yarn add`?

Reply:
https://classic.yarnpkg.com/en/docs/cli/add/

for example:

https://github.com/react-native-community/react-native-netinfo#getting-started

Reply:
Sorry, I fixed it @phongcao